security/backends/elasticsearch/models.py

```diff
@@ -135,6 +137,13 @@ class RequestLog(Log):
     state = EnumField(enum=RequestLogState)
     error_message = Text()
 
+    def __str__(self):
+        return ' '.join(
+            (force_text(v) for v in (
+                self.slug, self.response_code, localtime(self.start.replace(microsecond=0))
```

Comment:
this is wrong in inoutrequestlog you should use self.view_slug. Maybe will be better use short_path

Reply:
This is the same `__str__` I took from `RequestLog` in sql backend. AFAIK outputrequest log does not have view_slug. I can use short_path. This `__str__` will not be displayed in admin anyway, just in command line, because it will be overridden in `display_source` to access to verbose_name in core definition.

Reply:
so why write this method?

Reply:
for us its useless but consistent with sql backend. I have removed it.